src/collective/solr/upgrades.zcml

```diff
@@ -63,4 +63,14 @@
     profile="collective.solr:default"
     />
 
+  <genericsetup:upgradeStep
+    title="Upgrade to collective.solr 8"
+    description="Add stopwords registry entries"
+    source="7"
+    destination="8"
```

Comment:
@remdub thanks for your contribution! The current number in metadata.xml is 7:

https://github.com/collective/collective.solr/blob/main/src/collective/solr/profiles/default/metadata.xml#L3

Don't we need to upgrade that number as well then?